### ISSUE TITLE
Skipnav focus not working in IE

### ISSUE THREAD

**Issue by jasonkaedingICE** [18F/web-design-standards#1788]
## Description

We have been experiencing section 508 testing failures revolving around the skip link capability that is included in USWDS.  I took the opportunity to test out http://standards.usa.gov and found that in IE the skip link does not set focus properly either.  This led me to think that there is perhaps a systemic problem in the implementation.

In researching the issue, I found #482 (and related #519 to implement the fix for Chrome).  I'm wondering if this broke the behavior in IE, or whether it was something later on that may have broken it.

## Steps to reproduce the issue

1. Open IE (any version)
2. Visit http://standards.usa.gov
3. Click the flag image to set focus
4. Press Shift+Tab to access skip link
5. Press Enter (page should shift down)
6. Press Tab (focus returns back to skip link)

## Workarounds
If everything in the main-content has a tabindex attribute set, then this avoids the problem.  However, for complex pages it may be difficult to get the tabindex values right.

**Comment by juliaelman** [comment]
@jasonkaedingICE thank you for opening this issue. We have just completed sprint planning for this sprint and will add this to the backlog for our next sprint (3/27).

**Comment by shawnbot** [comment]
I see what [this fix](https://github.com/18F/web-design-standards/pull/519/files) does, and it seems to solve the problem, but I'm not sure if the JavaScript fix is the one we want. In our [accessibility guide](https://pages.18f.gov/accessibility/links/) (see: "Skip navigation"), we suggest setting `tabindex="-1"` on the target of the skip link, `#main-content`. I made this change in the IE11 inspector, and it worked.

@jasonkaedingICE, can you try adding `tabindex="-1"` to your main content element and let us know if that fixes the problem in your 508 tests?

**Comment by jasonkaedingICE** [comment]
I did try that out, and it does seem to work.  I'll have our 508 tester take a look tomorrow to confirm.

It would have been helpful if a mention of `tabindex="-1"` was also present in the accessibility section of the [header component page](https://standards.usa.gov/components/headers/). Were it so, I probably would have already attempted it, not knowing until now about this other accessibility guide.  Also, observing the failure on the USWDS site itself didn't give me a good example to emulate... ;)

**Comment by jasonkaedingICE** [comment]
An additional note: I did some further testing, and it looks like on some of our pages when we _do_ have explicit tabindex already set up for our content (and skipnav already was working), this change actually breaks those instances. Since this is part of our page template, I think we'll have to strip out tabindex from our pages completely, although for 508 reasons "natural" tab ordering should be the right way to go anyway.

**Comment by shawnbot** [comment]
Thanks, @jasonkaedingICE. We're mulling over options in [this pull request](https://github.com/18F/web-design-standards-docs/issues/230), so feel free to weigh in there if you have any thoughts on the implementation.

**Comment by jasonkaedingICE** [comment]
Our 508 tester confirmed that the `tabindex="-1"` fix is now compliant.  From my perspective, no code changes are required at all after we removed all other tabindex attributes from our application.

**Comment by shawnbot** [comment]
Cheers, and thanks for helping us figure it out @jasonkaedingICE!
